test/invoice_repository_test.rb

```diff
@@ -41,12 +41,12 @@ def test_it_finds_by_status
   end
 
   def test_finds_by_created_at
-    result = invoice_repository.find_by_created_at(DateTime.parse('2012-03-25 09:54:09 UTC'))
+    result = invoice_repository.find_by_created_at('2012-03-25 09:54:09 UTC')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.